docs/data-sources/management_access_policy.md

```diff
@@ -0,0 +1,175 @@
+  * `admin_state` (adminSt) - (string) The administrative state of the HTTP (Hypertext Transfer Protocol) Service.
+  * `annotation` (annotation) - (string) The annotation of the HTTP (Hypertext Transfer Protocol) Service object.
+  * `cli_only_mode` (cliOnlyMode) - (string) Determines if CLI-only mode is enabled.
+  * `description` (descr) - (string) The description of the HTTP (Hypertext Transfer Protocol) Service object.
+  * `global_throttle_rate` (globalThrottleRate) - (string) Maximum API calls allowed per unit time.
+  * `global_throttle_state` (globalThrottleSt) - (string) The throttle state for all clients without specific identifier in their headers.
+  * `global_throttle_unit` (globalThrottleUnit) - (string) The unit of measurement for rate limiting.
+  * `max_request_status_count` (maxRequestStatusCount) - (string) The maximum number of request status objects to keep.
+  * `name` (name) - (string) The name of the HTTP (Hypertext Transfer Protocol) Service object.
+  * `name_alias` (nameAlias) - (string) The name alias of the HTTP (Hypertext Transfer Protocol) Service object.
+  * `node_exporter` (nodeExporter) - (string) Node Exporter service status.
+  * `port` (port) - (string) The port used for HTTP communication service.
+  * `redirect_state` (redirectSt) - (string) The state of HTTP (Hypertext Transfer Protocol) communication service.
+  * `server_header` (serverHeader) - (string) Determines if the server header is included in responses.
+  * `throttle_rate` (throttleRate) - (string) The maximum login/refresh allowed per second.
+  * `throttle_state` (throttleSt) - (string) The state of login and refresh throttling.
+  * `visore_access` (visoreAccess) - (string) Grants permissions for accessing the Visore tool in Cisco ACI.
+    * `annotations` - (list) A list of Annotations (ACI object [tagAnnotation](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/tagAnnotation/overview)). This attribute is supported in ACI versions: 3.2(1l) and later.
+      * `key` (key) - (string) The key used to uniquely identify this configuration object.
+      * `value` (value) - (string) The value of the property.
+    * `tags` - (list) A list of Tags (ACI object [tagTag](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/tagTag/overview)). This attribute is supported in ACI versions: 3.2(1l) and later.
+      * `key` (key) - (string) The key used to uniquely identify this configuration object.
+      * `value` (value) - (string) The value of the property.
+* `http_ssl_configuration` - (map) A map of HTTP (Hypertext Transfer Protocol) SSL (Secure Sockets Layer) Configuration (ACI object [commHttps](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/commHttps/overview)).
```

Comment:
Perhaps HTTP and SSL only needs to be expanded in one of the options.
I think also in the format of "Hypertext Transfer Protocol (HTTP)" instead.

Reply:
Yes, I'm aware because I made a similar comment to a PR. However, in this case unless we add some unwanted logic we would need to stick with this. Let me give it a second look.
I made changes to the other half of your comment!

Reply:
Sounds good!

Reply:
@shrsr I have an idea for the abbreviations. What if we have a separate section in the markdown that just lists out the abbreviations. Then we don't need to use any full form terms in the option lists.
If that sounds okay I can add a separate task in the backlog for it?

Reply:
@samiib Yes that would be great!